resident/resident-service/pom.xml

```diff
@@ -1,533 +1,532 @@
 <?xml version="1.0"?>
 <project
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
-	xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>io.mosip.resident</groupId>
-		<artifactId>resident-parent</artifactId>
-		<version>1.2.1-SNAPSHOT</version>
-	</parent>
-	<artifactId>resident-service</artifactId>
-	<name>resident-service</name>
-	<version>1.2.1-SNAPSHOT</version>
-	<properties>
-		<commons.packet.manager.version>1.2.1-SNAPSHOT</commons.packet.manager.version>
-		<kernel.core.version>1.2.0.1-B1</kernel.core.version>
-		<kernel.virusscanner.clamav.version>1.2.1-SNAPSHOT</kernel.virusscanner.clamav.version>
-		<id-repository-core.version>1.2.0.1-SNAPSHOT</id-repository-core.version>
-		<kernel-keymanager-service.version>1.2.0.1-B1</kernel-keymanager-service.version>
-		<kernel.authcodeflowproxy.api.version>1.2.1-SNAPSHOT</kernel.authcodeflowproxy.api.version>
-		<kernel.parent.version>1.2.0</kernel.parent.version>
-		<kernel-cbeffutil-api.version>${kernel.parent.version}</kernel-cbeffutil-api.version>
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.mosip.resident</groupId>
+        <artifactId>resident-parent</artifactId>
+        <version>1.2.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>resident-service</artifactId>
+    <name>resident-service</name>
+    <version>1.2.1-SNAPSHOT</version>
+    <properties>
+        <commons.packet.manager.version>1.2.1-java21-SNAPSHOT</commons.packet.manager.version>
```

Comment:
Please update 1.2.1-java21-SNAPSHOT to 1.2.1-SNAPSHOT

Reply:
Done.